trackpy/linking.py

```diff
@@ -987,24 +1079,23 @@ def assign_candidates(cur_level, prev_hash, search_range, neighbor_strategy):
 class SubnetOversizeException(Exception):
     '''An :py:exc:`Exception` to be raised when the sub-nets are too big
     to be efficiently linked.  If you get this then either reduce your search range
-    or increase :py:attr:`sub_net_linker.MAX_SUB_NET_SIZE`'''
+    or increase :py:attr:`Linker.MAX_SUB_NET_SIZE`'''
     pass
 
 
-def recursive_linker_obj(s_sn, dest_size, search_range):
-    snl = sub_net_linker(s_sn, dest_size, search_range)
-    return zip(*snl.best_pairs)
+def recursive_linker_obj(s_sn, dest_size, search_range, max_size=30):
+    snl = sub_net_linker(s_sn, dest_size, search_range, max_size=max_size)
+    return [list(particles) for particles in zip(*snl.best_pairs)]
```

Comment:
These collections have to be mutable, and in Python 3 they are not unless I do something like this.